src/primitives/block.h

```diff
@@ -77,6 +77,8 @@ class CBlock : public CBlockHeader
     std::vector<CTransaction> vtx;
 
     // memory only
+    // TODO: cache check status inside the validation logic instead of inside of
+    // the primitive data type CBlock.
```

Comment:
This misses half the point of the comment - that we probably /do/ want to have the ability to cache the things that CBlock could check for itself (like the merkle tree).